apps/docs/content/components/Card.mdx

```diff
@@ -89,6 +89,44 @@ Combine `Icon` and `Label` when you need to add more metadata or context to the
 </Card>
 ```
 
+### Image
+
+Use the `Image` component to add an image to the `Card`. The `Image` component is optional and provides a visual context to the `Card`.
+
+```jsx live
+<Card href="https://github.com">
+  <Card.Image
+    src="https://via.placeholder.com/600x400/d3d9df/d3d9df.png"
+    alt=""
+    aspectRatio="16:9"
+  />
+  <Card.Heading>Code search & code view</Card.Heading>
+  <Card.Description>
+    Enables you to rapidly search, navigate, and understand code, right from
+    GitHub.com.
+  </Card.Description>
+</Card>
+```
+
+### Image and Label
+
+Combine `Image` and `Label` when you need to add more metadata or context to the `Card`. Use either the `Icon` or `Image` as the primary visual context, but not both.
```

Comment:
ooo interesting.. do we need to guardrail this in implementation then? I can do this straight in docs..

![Screenshot 2023-05-02 at 10 02 18](https://user-images.githubusercontent.com/13340707/235625007-97a5e235-babb-4480-97a0-7faab7688a5d.png)

I guess I'm asking how strict is this rule around icon + image? We typically trust users to _do the right thing_ but is that enough here?

Reply:
I thought about it too, I think we could release this by trusting users to compose the right Card. That and the upcoming design guidelines should help to mitigate wrong behaviours